templates/connect-inject-deployment.yaml

```diff
@@ -50,7 +53,7 @@ spec:
                 -tls-cert-file=/etc/connect-injector/certs/{{ .Values.connectInject.certs.certName }}
                 -tls-key-file=/etc/connect-injector/certs/{{ .Values.connectInject.certs.keyName }}
 {{- else }}
-                -tls-auto=consul-connect-injector-cfg \
+                -tls-auto={{ template "consul.fullname" . }}-connect-injector-cfg \
```

Comment:
The mutating webhook configuration name is actually set to `{{ template "consul.fullname" . }}-connect-injector-cfg` so I fixed it as part of this PR.

Reply:
It would be good to rebase on master to pick up the change to ${CONSUL_FULLNAME} from your other [PR](https://github.com/hashicorp/consul-helm/pull/39) so we don't overwrite that fix.